src/components/Datagrid/RowActions.tsx

```diff
@@ -0,0 +1,46 @@
+import { IconButton, MoreHorizontalIcon } from "@saleor/macaw-ui";
+import React from "react";
+
+import CardMenu, { CardMenuItem } from "../CardMenu";
+import useStyles from "./styles";
+
+interface RowActionsProps {
+  menuItems: CardMenuItem[];
+  disabled?: boolean;
+}
+
+export const RowActions = ({ menuItems, disabled }: RowActionsProps) => {
```

Comment:
Could you write a unit test for this component?

Reply:
the things you can test here are:
- rendering depending on thte `menuItems` prop (this seems like critical path)
- handling click and triggering onSelect 

also, a thing to consider - what I pass an empty array? perhaps would be nice to cover it to prevent some errors? or it's never the case?